optima/model.py

```diff
@@ -3,14 +3,14 @@
 from numpy import zeros, exp, maximum, minimum, hstack, inf, concatenate as cat
 from optima import OptimaException, printv, tic, toc, dcp, odict, makesimpars, Resultset
 
-def model(simpars=None, settings=None, verbose=2, safetymargin=0.8, benchmark=False, die=True):
+def model(simpars=None, settings=None, verbose=None, benchmark=False, die=True):
     """
-    This function runs the model. Safetymargin is how close to get to moving all people from a compartment in a single timestep.
+    This function runs the model. It's Optima, in other words.
```

Comment:
What about programs, scenarios, optimisations, GA?! I'd say this this is the epidemiological model within Optima, not Optima!